Extending/Property-Editors/value-converters.md

```diff
@@ -137,10 +137,9 @@ If all values should use the same level you can use the short cut below
 
             case PropertyCacheValue.XPath:
                 return PropertyCacheLevel.Content;
-
-            default:
```

Comment:
Why removing the default?

Reply:
do you think it's easier to read?

switch(...) {
  case ... :
   return ...
 default :
  return
}

vs.

switch(...){
  case ... :
   return ...
}
return ...

Reply:
It's not about easier to read.  A default does what it does.  You are able to change the code after the switch and change loose the missing enum. It is a change in future maintainability.  Anyway I reviewed the complete file (not only the change).  And it's good enough for me.  People won't be copying this without thinking anyway.